packages/flutter_chat/lib/src/screens/chat_detail/widgets/default_message_builder.dart

```diff
@@ -0,0 +1,293 @@
+import "package:cached_network_image/cached_network_image.dart";
+import "package:chat_repository_interface/chat_repository_interface.dart";
+import "package:flutter/material.dart";
+import "package:flutter_chat/src/config/chat_options.dart";
+import "package:flutter_chat/src/services/date_formatter.dart";
+import "package:flutter_chat/src/util/scope.dart";
+
+/// The standard side padding for the chat message
+const double _standardSidePadding = 20.0;
```

Comment:
Make this an option

Reply:
I created a spacing class that will have all the configurable paddings for the userstory